proto/umee/leverage/v1/tx.proto

```diff
@@ -35,39 +35,39 @@ service Msg {
   rpc Liquidate(MsgLiquidate) returns (MsgLiquidateResponse);
 }
 
-// MsgLendAsset represents a lender's request to lend a base asset type to the
+// MsgSupply represents a user's request to supply a base asset type to the
 // module.
```

Comment:
```suggestion
// MsgSupply is the request strucutre for the MsgLendAsset RPC.
```

Reply:
I think we can use it as a simpler form in other places as well. The RPC doc should be defined in the RPC method, not in the struct.